CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
```

Comment:
Perhaps make it a patch release so we install it right away?

Reply:
Ah no, there is also the other PR merged. We can create a patch branch in another PR, never mind